eng/scripts/helpers/Changelog-Helpers.ps1

```diff
@@ -0,0 +1,531 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+    Helper functions for changelog automation.
+
+.DESCRIPTION
+    This file provides helper functions for automating changelog updates by comparing
+    JAR files and updating CHANGELOG.md files.
+#>
+
+function Get-MavenArtifactInfo {
```

Comment:
Can we re-use the existing function in the `language-settings.ps1`?
https://github.com/Azure/azure-sdk-for-java/blob/1406557fa6115c36647fa6d8a91abf0ff25a5f04/eng/scripts/Language-Settings.ps1#L323

Reply:
As Shanghai side plans to finish the V1 scenario related work in this week. To unblock merge this PR, first resolve this comment, and will have a follow up PR as it takes time to investigate. But let's keep discussing on the perf issue above. Thanks.